automated/linux/armnn/armnn.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+OUTPUT_FILE="${OUTPUT}/results.txt"
+PYTEST="false"
+ARMNN_TARBALL="https://snapshots.linaro.org/components/armnn/latest/armnn-full.tar.xz"
+SKIP_INSTALL="false"
+TEST_DIR="/tmp/armnn"
+UNIT_TESTS="true"
+
+usage() {
+  echo "Usage: $0 [-s <true|false>]
+                  [-a <armnn-tarball>]
+                  [-t <true|false>]
+                  [-p <true|false>]
+                  [-d <testing-dir-location>]" 1>&2
+  exit 1
+}
+
+while getopts "s:a:t:p:d:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    a) ARMNN_TARBALL="${OPTARG}" ;;
+    t) UNIT_TESTS="${OPTARG}" ;;
+    p) PYTEST="${OPTARG}" ;;
+    d) TEST_DIR="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+pkgs="ntp wget gcc g++ systemd"
+if [ "${PYTEST}" = true ]; then
+  pkgs2="python3-dev python3-pip"
+fi
+dhclient
+install_deps "${pkgs}" "${SKIP_INSTALL}"
+
+systemctl restart ntp
+systemctl enable ntp
```

Comment:
Would it possible to stop noise on the console by re-directing output to /dev/null ?
systemctl enable ntp 2>/dev/null